hub/constants.py

```diff
@@ -36,8 +36,12 @@
     for c in SUPPORTED_COMPRESSIONS
     if c.upper() in Image.SAVE and c.upper() in Image.OPEN
 ]
+
+SUPPORTED_COMPRESSIONS.append("lz4")
```

Comment:
can we have 3 constants instead of just 1:

- `SAMPLE_COMPRESSORS`
- `CHUNK_COMPRESSORS`
- `ALL_COMPRESSORS = list(set(SAMPLE_COMPRESSIONS + CHUNK_COMPRESSIONS))`